source/composer.apib

```diff
@@ -32,6 +38,14 @@ API Blueprint as defined in its [specification](https://github.com/apiaryio/api-
 
 + Relation: compose
 
++ Request Compose API Blueprint from API Elements API (application/vnd.refract+json)
+
+        :[](fixtures/apib/normal.refract.json)
```

Comment:
This file does not exist, did you mean `fixtures/apib/normal.refract.parse-result.json`?